### ISSUE TITLE
NTR: Temporal predominance to brain imaging abnormality (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8933]
New term request Temporal predominance to brain imaging abnormality (elthat_101022000701): Dysmyelination; white matter; FTLD  UMLS: 0442043

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000850
- label: Temporal cortex localization
- parents: [Brain localization (HP:0430127)](https://hpo.jax.org/app/browse/term/HP:0430127)
- pmid: 
- def: Localization to or predominance in the temporal cortex of the brain.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8933
- synonyms:
